docs/src/index.rst

```diff
@@ -13,7 +13,6 @@
    :caption: User's Guide
    :maxdepth: 1
 
-   user/pytorch
```

Comment:
This is causing an error.

https://travis-ci.org/mila-udem/orion/jobs/418260617#L571

 I would keep this section there even though it is somehow a duplicate.